src/modules/python/libkvipython.cpp

```diff
@@ -502,13 +502,39 @@ static bool python_kvs_fnc_isAvailable(KviKvsModuleFunctionCall * c)
 	return true;
 }
 
+/*
+	@doc: python.version
+	@type:
+		function
+	@title:
+		$python.version
+	@short:
+		Check which version of Python is supported in this build of KVIrc
+	@syntax:
+		$python.version
+	@description:
+		Returns which major version of Python is KVIrc linked to ([b]2[/b] or [b]3[/b])
+		or [b]0[/b] if Python is not supported at all.
+*/
+
+static bool python_kvs_fnc_version(KviKvsModuleFunctionCall * c)
+{
+#ifdef COMPILE_PYTHON_SUPPORT
+	c->returnValue()->setInteger(PY_MAJOR_VERSION);
```

Comment:
Maybe value containing also at least the minor version of Python would be more useful.
Python headers, beside `PY_MAJOR_VERSION`, `PY_MINOR_VERSION`, `PY_MICRO_VERSION`, `PY_RELEASE_LEVEL`, `PY_RELEASE_SERIAL`, already provide `PY_VERSION_HEX`:
```
/* Version as a single 4-byte hex number, e.g. 0x010502B2 == 1.5.2b2.
   Use this for numeric comparisons, e.g. #if PY_VERSION_HEX >= ... */
#define PY_VERSION_HEX ((PY_MAJOR_VERSION << 24) | \
                        (PY_MINOR_VERSION << 16) | \
                        (PY_MICRO_VERSION <<  8) | \
                        (PY_RELEASE_LEVEL <<  4) | \
                        (PY_RELEASE_SERIAL << 0))
```